projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-top-posts.php

```diff
@@ -0,0 +1,201 @@
+<?php
```

Comment:
The endpoint needs to be wired up for WPCOM, which I can help with when the PR is merged.